### PR TITLE
fix: Support non-semantic Rasa X image tag

### PR DESCRIPTION
**Pull request by HotThoughts**
- Skip semantic version checking when `rasa-x.tag` is `master`, `latest`, or has `a` or `rc` in it
- Bump `chart-testing-action` version v1.0.0 -> v2.0.1
- Update python version from 3.6.9 to 3.7 in CI workflow since it is required by the `chart-testing-action@v2.0.1`

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
 
     - name: Run chart-testing (lint)
       id: lint
-      uses: helm/chart-testing-action@v1.0.0
+      uses: helm/chart-testing-action@b0d4458c71155b54fcf33e11dd465dc923550009
```
Comment (tczekajlo) [inline .github/workflows/ci.yml:36]: The action fails

```
Requirement already satisfied: pyyaml in /opt/hostedtoolcache/ct/v3.3.0/x86_64/venv/lib/python3.6/site-packages (from yamale==3.0.4)
Building wheels for collected packages: yamale
  Running setup.py bdist_wheel for yamale: started
  Running setup.py bdist_wheel for yamale: finished with status 'error'
  Complete output from command /opt/hostedtoolcache/ct/v3.3.0/x86_64/venv/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-c2y5j063/yamale/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/tmpgjeo_gsgpip-wheel- --python-tag cp36:
  /usr/lib/python3.6/distutils/dist.py:261: UserWarning: Unknown distribution option: 'long_description_content_type'
    warnings.warn(msg)
  usage: -c [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: -c --help [cmd1 cmd2 ...]
     or: -c --help-commands
     or: -c cmd --help
  
  error: invalid command 'bdist_wheel'
  
  ----------------------------------------
  Running setup.py clean for yamale
  Failed building wheel for yamale
```

**Review by tczekajlo — CHANGES_REQUESTED** [review]
You have to bump up the chart version

File: .github/workflows/ci.yml (hunk 1 above)
Comment (HotThoughts) [reply .github/workflows/ci.yml:42]: Fixed it by updating python version to 3.7😉

Comment (HotThoughts) [reply .github/workflows/ci.yml:42]: Reference: https://github.com/helm/chart-testing-action/issues/65

**Comment by tczekajlo** [comment]
@HotThoughts You have to bump up the Chart version otherwise it won't release

**Comment by HotThoughts** [comment]
> @HotThoughts You have to bump up the Chart version otherwise it won't be released - https://github.com/RasaHQ/rasa-x-helm/blob/main/charts/rasa-x/Chart.yaml#L4

Oops. Thanks for keeping reminding me. I now finally understand what it means by bump chart version. 😀

**Review by tczekajlo — APPROVED** [review]
LGTM

**Pull request merged**